code-build.js

```diff
@@ -64,11 +89,12 @@ async function waitForBuildEndTime(sdk, { id, logs }, nextToken) {
   // We did it! We can stop looking!
   if (current.endTime && !events.length) return current;
 
-  // More to do: Sleep for 5 seconds :)
+  // More to do: Sleep for a few seconds to avoid rate limiting
   await new Promise(resolve => setTimeout(resolve, wait));
 
   // Try again
   return waitForBuildEndTime(sdk, current, nextForwardToken);
+
```

Comment:
nit: As long as we are working on the error,
lets kill the whitespace here :)
(clean diff ftw)